projects/core/src/user/store/reducers/user-orders.reducer.ts

```diff
@@ -1,44 +1,25 @@
-import * as fromUserOrdersAction from '../actions/user-orders.action';
 import * as fromAction from '../actions/index';
-import { UserOrdersState } from '../user-state';
-import { OrderHistoryList } from '../../../occ/occ-models';
+import * as fromUserOrdersAction from '../actions/user-orders.action';
+import { OrderHistoryList } from '../../../occ';
 
-export const initialState: UserOrdersState = {
-  orders: {
-    orders: [],
-    pagination: {},
-    sorts: []
-  },
-  loading: false,
-  loaded: false
+export const initialState: OrderHistoryList = {
+  orders: [],
+  pagination: {},
+  sorts: []
 };
 
 export function reducer(
   state = initialState,
   action: fromUserOrdersAction.UserOrdersAction | fromAction.MiscsDataAction
-): UserOrdersState {
+): OrderHistoryList {
   switch (action.type) {
-    case fromUserOrdersAction.LOAD_USER_ORDERS: {
-      return {
-        ...state,
-        loaded: false,
-        loading: true
-      };
-    }
     case fromUserOrdersAction.LOAD_USER_ORDERS_SUCCESS: {
       const orders: OrderHistoryList = action.payload;
```

Comment:
this `case` can be removed. the action.payload will be set to value by default.

Reply:
Not quite: as we are combining this reducer with the loader reducer, the loader reducer will call this reducer to handle the success and error actions.